protocols/rest-jakarta/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Parent -->
+  <parent>
+    <groupId>org.jboss.arquillian</groupId>
+    <artifactId>arquillian-build</artifactId>
+    <version>1.7.0.Final-SNAPSHOT</version>
+    <relativePath>../../build/pom.xml</relativePath>
+  </parent>
+
+  <!-- Model Version -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Artifact Configuration -->
+  <groupId>org.jboss.arquillian.protocol</groupId>
+  <artifactId>arquillian-protocol-rest-jakarta</artifactId>
+  <name>Arquillian Protocol REST 3.x</name>
+  <description>Protocol handler for communicating using REST following the Jakarta RESTful Web Services 3.x spec and jakarta.ws.rs.* apis
+  </description>
+
+  <!-- Properties -->
+  <properties>
+    <!-- Versioning -->
+    <version.jetty_jetty>11.0.5</version.jetty_jetty>
+    <version.resteasy>6.0.3.Final</version.resteasy>
+    <version.restfulws-api>3.0.0</version.restfulws-api>
+    <!-- The protocol adaptor requires Java 8 -->
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Jakarta EE 9 supports Java 8 and Java 11, so that is why I left this as Java 8 so it can be used for Jakarta EE 9 testing.  Note I have a dependency on Jakarta RESTful Web Services 3.0 which is EE9.